tests/specs/tram-spec.js

```diff
@@ -71,6 +72,18 @@ const tests = (Tram) => describe('Tram', () => {
     })
   })
 
+  describe('addListener', () => {
+    it('should call single listener', () => {
+      const spy = jasmine.createSpy('listener')
+      const app = new Tram()
+      app.addActions({counter: counterActions})
+      app.addListener(spy)
+      app.engine.actions.add()
```

Comment:
instead of `app.engine.actions.add()` we want the listener to be called from Tram-One (right now this really only tests that hover-engine works).

Instead, we should have a page that triggers an action when the app is started.

Reply:
```
app.addActions({actions: switchAction})
app.addListener(spy)
pageWithAction(app.engine.store, app.engine.actions)

expect(spy).toHaveBeenCalled()
```
```
const pageWithAction = (store, actions) => {
    const html = Tram.html()
    if (store) { actions.toggle() }
    return html`<div>${store}</div>`
}
const switchAction = {
    init: () => true,
    toggle: () => false
}
```
Something like this ?

Reply:
Yeah! But, instead of `pageWithAction(app.engine.store, app.engine.actions)` we'll want to do the same thing we do with the other tests, and actually call `app.start`